store-implementation/proxy-store/src/test/java/uk/gov/gchq/gaffer/proxystore/integration/ProxyStoreResponseDeserialiserIT.java

```diff
@@ -89,16 +95,30 @@ public void shouldUseOperationsResponseDeserialiserToDeserialiseOperationsRespon
                 .store(proxyStore)
                 .build();
 
-        verify(operationResponseDeserialiser).deserialise(anyString());
-
         final Set<Class<? extends Operation>> actualOperationClasses = proxyStore.getSupportedOperations();
         final Set<Class<? extends Operation>> expectedOperationClasses = new HashSet<>(storeOperations);
         expectedOperationClasses.add(OperationChain.class);
         expectedOperationClasses.add(OperationChainDAO.class);
         expectedOperationClasses.add(GetTraits.class);
 
-        assertEquals(actualOperationClasses.size(), expectedOperationClasses.size());
-        assertTrue(actualOperationClasses.containsAll(expectedOperationClasses));
+        //Because of Graph.updateGraphHooks the hook resolvers are forced in, requiring these Handlers
+        // Named operation
+        expectedOperationClasses.add(NamedOperation.class);
+        expectedOperationClasses.add(AddNamedOperation.class);
+        expectedOperationClasses.add(GetAllNamedOperations.class);
+        expectedOperationClasses.add(DeleteNamedOperation.class);
+
+        // Named view
+        expectedOperationClasses.add(AddNamedView.class);
+        expectedOperationClasses.add(GetAllNamedViews.class);
+        expectedOperationClasses.add(DeleteNamedView.class);
+
+        Assertions.assertThat(actualOperationClasses)
+                .containsExactlyInAnyOrderElementsOf(expectedOperationClasses)
+                //This is actually what is getting inserted via the mock during a fetchOperations()
+                .contains(AddElements.class);
```

Comment:
It doesn't look like `expectedOperationClasses` contains `AddElement`, why is this a separate `contains`?

Reply:
AddElements comes from the Mock higher up in the test. 
And that is kinda what the test is actually checking for.
Its a poor test, I didn't write it.
It required extra handlers added due to graph.hooks being forced in, this gets fixed in a later PR.